site/app/templates/admin/Report.twig

```diff
@@ -64,5 +64,33 @@
                 </button>
             </div>
         </div>
+        <hr>
+        <div>
+            <div id="grade-report-customization-upload" class="card">
+                <div class="title-cont">
+                    Upload a Rainbow Grades customization.json directly to Submitty
+                </div>
+                <div class="card-btn-cont">
+                    <form action="http://localhost:1511/courses/f21/sample/reports/rainbow_grades_customization/upload" method="POST" enctype="multipart/form-data">
+                        <input type="hidden" name="csrf_token" value="{{csrfToken}}">
+                        <input type="file" id="config-upload" name="config_upload" style="display:none">
+                    
+                            <button onclick="promptCustomizationUpload.call(this)"
+                                    class="btn btn-primary key_to_click"
+                                    tabindex="0"
+                                    type="button"
+                            >
```

Comment:
Usual recommended style for these sorts of blocks is:
```suggestion
                            <button
                                onclick="promptCustomizationUpload.call(this)"
                                class="btn btn-primary key_to_click"
                                tabindex="0"
                                type="button"
                            >
```

Similar to if you had a long function or if statement, where the opener/closer are on its own line separate from the arguments.